go.mod

```
@@ -3,16 +3,41 @@ module github.com/fossology/LicenseDb
 go 1.18
```

```suggestion
go 1.20
```
go 1.18 has already reached EOL, please update your installation and then update this mod file.